### PR TITLE
Minor typo fixes

### PR DESCRIPTION
**Pull request by Bertie690**
Just some minor stuff I thought to fix. 
:)

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
     "files.exclude": {
-        "frontend": true
+        "frontend": false
```
Comment (sirgwain) [inline .vscode/settings.json]: Please revert this change. It's in the docs to open the workspace file.

https://github.com/sirgwain/craig-stars/blob/develop/docs/development.md#visual-studio-code

**Review by sirgwain — CHANGES_REQUESTED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (Bertie690) [reply .vscode/settings.json]: k

**Review by sirgwain — APPROVED** [review]
